src/AuthForm.jsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
```

Comment:
Nice work keeping all auth logic in one component and either showing the form to login or the button ton sign out, really smart UI and UX thought!